app/controllers/appeal-application/fee-support.ts

```diff
@@ -0,0 +1,59 @@
+import { NextFunction, Request, Response, Router } from 'express';
+import _ from 'lodash';
+import { FEATURE_FLAGS } from '../../data/constants';
+import { paths } from '../../paths';
+import LaunchDarklyService from '../../service/launchDarkly-service';
+import UpdateAppealService from '../../service/update-appeal-service';
+
+function getFeeSupportQuestion(appeal: Appeal) {
+  let hint: string;
+  let decision: string;
+  hint = '';
+  const question = {
+    title: 'title',
+    hint,
+    options: [
+      {
+        value: '',
+        text: '',
+        checked: decision === ''
+      },
+      {
+        value: '',
+        text: '',
+        checked: decision === ''
+      }
+    ],
+    inline: false
+  };
+  return question;
+}
+
+async function getFeeSupport(req: Request, res: Response, next: NextFunction) {
+  try {
+    const paymentsFlag = await LaunchDarklyService.getInstance().getVariation(req, FEATURE_FLAGS.CARD_PAYMENTS, false);
```

Comment:
This payments feature toggle has been on since December 2021. Not for you to change Miguel but there should be a tech debt ticket to remove it throughout the code.

Reply:
Speaking with @ikirsanov  on his ticket - remove the payments flag for new pages. No need to remove completely on pages that you have not touched.

Reply:
I already did the changes please have a look again